streams/runvlc.py

```diff
@@ -32,8 +32,16 @@
 import json
 import vlc
 import argparse
+import signal
 from typing import List, Optional, Any, IO
 
+
+def signal_handler(sig, _):
+  """Handle sigterm signal."""
+  log(f"Caught signal {sig}, exiting.")
+  sys.exit(0)
```

Comment:
does this actually kill the underlying VLC proc? [i don't see a `__del__()` method on the Instance class](https://www.olivieraubert.net/vlc/python-ctypes/doc/vlc.Instance-class.html) but perhaps this is being handled in another, ooky spooky way by the VLC lib (or I don't understand it well enough.)

Reply:
ah, this pokes `libvlc`, it doesn't create an underlying vlc proc. that works for me